src/routes/admin/screenmessage/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { pb } from "$lib/stores/authStore";
+
+  export let data;
+  let screenMessages = data.screenMessages;
+  let activeMessage = data.activeMessage[0];
+
+  let open = !activeMessage.isVisible;
+  async function updateActiveMessage() {
+    await pb.collection("activeMessage").update(activeMessage.id, {
+      message: selectedMessage.id,
+      isVisible: !open
+    });
+  }
+
+  let selectedMessage;
```

Comment:
Trenger man egentlig to ulike variabler for `selectedMessage` og `activeMessage`? Man kan vel beholde samme funksjonalitet med kun en av dem.

Reply:
SelectedMessage er en `screenMessage` ikke en `activeMessage` som har en relation til en `screenMessasge`. Dette er virkelig forvirrende navn, men jeg vet ikke hva de burde være.

Reply:
> SelectedMessage er en `screenMessage` ikke en `activeMessage` som har en relation til en `screenMessasge`. Dette er virkelig forvirrende navn, men jeg vet ikke hva de burde være.

Det gir mer mening. Du kan kanskje legge til en kommentar som forklarer det bedre. Hvis vi får pocketbase types til å virke i fremtiden, vil det bli enda enklere å forstå det.

Reply:
Har prøvd å endre navn på noen variabler for å forklare bedre.